sceptre/sageit/templates/admodeladexplorer-redirector.yaml

```diff
@@ -0,0 +1,150 @@
+        Origins:
+          - DomainName: !Sub
+            - ${stackPrefix}.admodeladexplorer.org
+            - {stackPrefix: !Ref StackType}
+            Id: S3Origin
+            CustomOriginConfig:
+              HTTPPort: 80
+              HTTPSPort: 443
+              OriginProtocolPolicy: http-only
+        Enabled: true
+        HttpVersion: 'http2'
+        DefaultRootObject: index.html
+        Aliases:
+          - !Sub
+            - ${stackPrefix}.admodelexplorer.org
+            - { stackPrefix: !Ref StackType }
+          - !Sub
+            - ${stackPrefix}.modeladexplorer.org
+            - { stackPrefix: !Ref StackType }
+          - !If
+            - IsProdEnv
+            - admodelexplorer.org
+            - !Ref AWS::NoValue
+          - !If
+            - IsProdEnv
+            - modeladexplorer.org
+            - !Ref AWS::NoValue
+        CustomErrorResponses:
+          - ErrorCachingMinTTL: 60
+            ErrorCode: 404
+            ResponseCode: 200
+            ResponsePagePath: '/index.html'
+          - ErrorCachingMinTTL: 60
+            ErrorCode: 403
+            ResponseCode: 200
+            ResponsePagePath: '/index.html'
+        DefaultCacheBehavior:
+          DefaultTTL: 3600
+          AllowedMethods:
+            - GET
+            - HEAD
+          Compress: true
+          TargetOriginId: S3Origin
+          ForwardedValues:
+            QueryString: true
+            Cookies:
+              Forward: none
+          FunctionAssociations:
+          -
+            EventType: viewer-request
+            FunctionARN: !Sub
+            - arn:aws:cloudfront::797640923903:function/${stackPrefix}-modeladexplorer-redirector-fct
+            - { stackPrefix: !Ref StackType }
+          ViewerProtocolPolicy: redirect-to-https
+        PriceClass: PriceClass_100
+        ViewerCertificate:
+          AcmCertificateArn: !Ref AcmCertificateArn
+          SslSupportMethod: sni-only
+  RedirectFct:
+    Type: AWS::CloudFront::Function
+    Properties:
+      AutoPublish: true
+      FunctionCode: |
+        function handler(event) {
+          var req = event.request;
+          var host = req.headers.host.value;
+          var response = {statusCode: 307, statusDescription: 'OK', headers: { 'cloudfront-functions': { 'value': 'generated-by-CloudFront-Functions' }}}
+          if (host.includes('admodel')) {
+            response['headers']['location'] = { 'value': 'https://modeladexplorer.org' };
+          } else {
+            response['headers']['location'] = { 'value': 'https://shinypro.synapse.org/modeladexplorer/' };
+          }
+        return response;
+        }
+      FunctionConfig:
+        Comment: If coming as admodel, redirect to modelad else redirect to shiny
+        Runtime: cloudfront-js-1.0
+      FunctionMetadata:
+        FunctionARN: !Sub
+        - arn:aws:cloudfront::797640923903:function/${stackPrefix}-modeladexplorer-redirector-fct
```

Comment:
can we use a ${AWS::AccountId} instead of hard coding the account id?

Reply:
Done